.github/workflows/fdroid-publish.yml

```
@@ -0,0 +1,104 @@
+      - name: Update F-Droid repository
+        run: |
+          cd fdroid
+          fdroid update --create-metadata --config config.yml
+
+      - name: Clean sensitive files
+        if: always()
+        run: |
+          rm -f fdroid/keystore.jks fdroid/config.yml
+
+      - name: Publish repository to gh-pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)